tests/integration/test_charm.py

```diff
@@ -47,3 +51,40 @@ async def test_connectivity(ops_test: OpsTest):
     appurl = f"http://{address}:80/"
     r = requests.get(appurl)
     assert r.status_code == 200
+
+
+async def test_integrate_blackbox(ops_test: OpsTest):
+    # @todo: upgrade to stable when blackbox charm with probes relation
+    # is promoted from edge.
+    await ops_test.model.deploy(
+        "blackbox-exporter-k8s", "blackbox", channel="latest/edge", trust=True
```

Comment:
How about using 'stable' ?

Reply:
blackbox with probes relation is currently in edge, added to do to upgrade to stable in the future.